src/features/DebugSession.ts

```diff
@@ -480,6 +484,17 @@ export class DebugSessionFeature extends LanguageClientConsumer
             }
         }
 
+        // NOTE: We don't support attaching to the Extension Terminal, even
+        // though in the past it looked like we did. The implementation was
+        // half-baked and left things unusable.
+        if (config.processId === "current" || config.processId === await this.sessionManager.getLanguageServerPid()) {
+            // TODO: When (if ever) it's supported, we need to convert 0 and the
+            // old notion of "current" to the actual process ID, like this:
+            // config.processId = await this.sessionManager.getLanguageServerPid();
+            void this.logger.writeAndShowError("Attaching to the PowerShell Extension terminal is not supported. Please use the 'PowerShell: Interactive Session' debug configuration instead.");
```

Comment:
A great UX you provided for probably noone using it at all :)

Reply:
I sure hope that's the case. It's crazy to me that we've literally been shipping provided debug config that doesn't work, and never did!